Protocol.md

```diff
@@ -627,6 +631,13 @@ subprotocol strings, and:
   SHALL validate that the negotiated subprotocol is present in the
   *subprotocols* field.
 
+Furthermore, the server SHALL validate that the *ping_interval* field
+contains a non-negative integer. If the value is `0`, the server SHALL
+NOT send WebSocket *ping* messages to the client. Otherwise, the server
+SHOULD send a WebSocket *ping* message in the requested interval to the
+client. A timeout of 30 seconds for unanswered *ping* messages is
+RECOMMENDED. An unanswered *ping* shall result in a protocol error.
```

Comment:
Again, that would be unwanted behaviour as an unanswered *ping* message MUST be treated as a protocol error. I'll switch the sentences though.